### ISSUE TITLE
New-Mountain-map

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#167]
```
hi, so i´ve made 3 maps...
so if you like, take a look at it and tell me if this hits the sense of 
andor´s trail. and if you want me so, i would really like to build a complete 
mountain-area....
```

Original issue reported on code.google.com by `michisch...@web.de` on 2 Mar 2011 at 6:28

Attachments:
- [blackwater_mountain1.tmx](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-0/blackwater_mountain1.tmx)
- [blackwater_mountain2.tmx](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-0/blackwater_mountain2.tmx)
- [blackwater_mountain3.tmx](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-0/blackwater_mountain3.tmx)

**Comment by GoogleCodeExporter** [comment]
```
They are a good start.
```

Original comment by `sdeva...@gmail.com` on 3 Mar 2011 at 1:30
- Changed title: **New-Mountain-map**
- Added labels: **Type-Enhancement**, **Priority-Low**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
thanks, a lott, i´m trying to improve them and make some new. when it´s done 
i´m going to post the updated version...
```

Original comment by `michisch...@web.de` on 3 Mar 2011 at 9:54

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Some of maps have a white part at the top edge of the tiles - can this not be 
prevented ?
It kinda breaks the terrain.

Map6 has sharp edge near bottom exit. Also, I think having just a small passage 
way makes it unneeded.

If we are to break the fallen rocks, then we need some way to blow them up, 
like throwing or planting a device.

I like the dark maps with "gem light", however our character should emit some 
light - maybe 2 titles radius.
```

Original comment by `kims...@gmail.com` on 4 Mar 2011 at 12:31

**Comment by GoogleCodeExporter** [comment]
```
thanks, everything is fixed now.
```

Original comment by `michisch...@web.de` on 4 Mar 2011 at 5:41

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Hm, i've just recognized that oskar wants a mountain called galmore. If he 
likes my maps i could change the name from blackwater mountain into mount 
galmore. I also thought about a content of around 40 single maps when it's done.
So question @ oskar: do you like my look and want me to buil mount galmore? But 
there are 3 things i can't do: 
- friendly npcs and the texts they are saying
- quests and story (it's your plot, not mine)
- boss npcs
Btw: i'm doing my best to be ready at the 0.7 release...
```

Original comment by `michisch...@web.de` on 4 Mar 2011 at 9:37

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
wow i love the maps above. i hope Oskar could add that to the game, and create 
a few quests around it.
```

Original comment by `jesscahi...@gmail.com` on 5 Mar 2011 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
Nice maps. Well done michisch...

I have added some comments to the maps on the attached images. This could 
really be something that we can add later on.

Some questions for you to think about:
 - How do these maps fit together to each other?
 - What types of monsters or NPCs did you have in mind for them? 
 - Where on the worldmap do you think these maps would fit?
      http://www.iostr.com/andorstrail_worldmap.png

Overall, I think they look good. I would be happy to add these in some way to 
the next version. We should also think about what dialogue and NPCs to add to 
them.

Well done! Looking forward to seeing them improve even more!
```

Original comment by `oskar.wi...@gmail.com` on 6 Mar 2011 at 8:41

Attachments:
- [blackwater_mountain3.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-12/blackwater_mountain3.png)
- [blackwater_mountain4.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-12/blackwater_mountain4.png)
- [blackwater_mountain5.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-12/blackwater_mountain5.png)
- [blackwater_mountain6.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-12/blackwater_mountain6.png)
- [blackwater_mountain7.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-12/blackwater_mountain7.png)
- [blackwater_mountain8.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-12/blackwater_mountain8.png)
- [blackwater_mountain9.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-12/blackwater_mountain9.png)
- [blackwater_mountain10.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-12/blackwater_mountain10.png)
- [blackwater_mountain11.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-12/blackwater_mountain11.png)
- [blackwater_mountain12.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-12/blackwater_mountain12.png)
- [blackwater_mountain13.png](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-167/comment-12/blackwater_mountain13.png)

**Comment by GoogleCodeExporter** [comment]
```
hi oskar,
wow that great! i really look foreward to make new maps for the game.
i already improved the old ones (the things you disliked).
i´m going to post again, when i´ve more time, but i found a place for my 
mountain:
the entry is just in the middle between flagstone and stoutford.
the mountain takes the hole grey space in the north of stoutford...
```

Original comment by `michisch...@web.de` on 6 Mar 2011 at 11:59

**Comment by GoogleCodeExporter** [comment]
```
Well, just finished map 25 of 50.
So blackwater mountain is 50% completed.

Pm: the single parts in dark map (7 and 8) are connected. I wanted something 
"pitchblack styled"...
To the quests: i already have the basic idea, but there is a lot more to come. 
When i am ready i will send them directly to oskar (no spoilers allowed ;-) ). 
Just give you a hint:
Blackwater mountain is the place where the great lord gets his gold from. So of 
course there might be some people interrested in cutting this source of in 
order to weaken the great lord... ( and no, there won't be a 50k gold loot for 
you ;-) )

@oskar: this a complete and connected mountainchain. You just have to add the 
npcs and help me with the quests. Then just connet flakstone0_west (direction 
to stoutford) with blackwater_mountain0_east to add all the 50 maps!
```

Original comment by `michisch...@web.de` on 6 Mar 2011 at 4:32

**Comment by GoogleCodeExporter** [comment]
```
...i´ve deleted my privious comments, because oskar has postet my maps, and 
they do not need to show up everywhere ;)
```

Original comment by `michisch...@web.de` on 7 Mar 2011 at 6:57

**Comment by GoogleCodeExporter** [comment]
```
These maps are now included in svn. They will be included in the next update.
```

Original comment by `oskar.wi...@gmail.com` on 22 Mar 2011 at 6:21
- Changed state: **Ready**
- Added labels: **Milestone-0.6.9**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:50
- Changed state: **Done**
